eslint-config/rules/plugin-jsdoc.js

```diff
@@ -0,0 +1,247 @@
+module.exports = {
+  plugins: [
+    'jsdoc',
+  ],
+  rules: {
+    /**
+     * Checks that `@access` tags are valid.
+     */
+    'jsdoc/check-access': ['off'],
+
+
+    /**
+     * Reports invalid alignment of JSDoc block asterisks.
+     */
+    'jsdoc/check-alignment': ['warn'],
+
+
+    /**
+     * Ensures that (JavaScript) examples within JSDoc adhere to ESLint rules.
+     */
+    'jsdoc/check-examples': ['warn'],
+
+
+    /**
+     * Reports invalid padding inside JSDoc blocks.
+     */
+    'jsdoc/check-indentation': ['warn'],
+
+
+    /**
+     * Ensures that parameter names in JSDoc match those in the function declaration.
+     */
+    'jsdoc/check-param-names': ['warn'],
+
+
+    /**
+     * Ensures that property names in JSDoc are not duplicated on the same block and that nested properties have defined roots.
+     */
+    'jsdoc/check-property-names': ['off'],
+
+
+    /**
+     * Reports against Google Closure Compiler syntax.
+     */
+    'jsdoc/check-syntax': ['off'],
+
+
+    /**
+     * Reports invalid block tag names.
+     */
+    'jsdoc/check-tag-names': ['warn', {
+      definedTags: ['endpoint'],
+    }],
+
+
+    /**
+     * Reports invalid value types.
+     */
+    'jsdoc/check-types': ['warn'],
+
+
+    /**
+     * Checks the values of specific metadata tags.
+     *
+     * https://github.com/gajus/eslint-plugin-jsdoc/blob/master/README.md#check-values
+     */
+    'jsdoc/check-values': ['off'],
+
+
+    /**
+     * Expects specific tags to be empty of any content.
+     *
+     * https://github.com/gajus/eslint-plugin-jsdoc/blob/master/README.md#empty-tags
+     */
+    'jsdoc/empty-tags': ['off'],
+
+
+    /**
+     * Reports an issue with any non-constructor function using @implements.
+     */
+    'jsdoc/implements-on-classes': ['off'],
+
+
+    /**
+     * Enforces a regular expression pattern on descriptions.
+     */
+    'jsdoc/match-description': ['off'],
+
+
+    /**
+     * Enforces a consistent padding of the block description.
+     */
+    'jsdoc/newline-after-description': ['off'],
+
+
+    /**
+     * Checks for multi-line-style comments which fail to meet the criteria of a jsdoc block.
+     */
+    'jsdoc/no-bad-blocks': ['off'],
+
+
+    /**
+     * Reports defaults being used on the relevant portion of `@param` or `@default`.
+     */
+    'jsdoc/no-defaults': ['off'],
```

Comment:
This is another new rule I found while digging around in the source (turns out there is also another .readme directory with docs for this and `no-bad-blocks`. I was just blind and only looked at their main README. ಠ_ಠ

Not sure if we want it though.

https://github.com/gajus/eslint-plugin-jsdoc/blob/master/.README/rules/no-defaults.md

Reply:
I don't see why we would use this rule?

Reply:
I believe it's because one of the major parsers prefers to interpret defaults by parsing the arguments themselves. This isn't in all parsers though so I don't think its of use to us.

Reply:
in all my functions with optional arguments, they are defined using the [name] syntax, am I to understand this rule would disallow that

Reply:
yeah it would. lets leave it off then.